### DISCUSSION TITLE
大数计算出错，当p为负数时，mod运算结果不正确。不应该是负数。

### DISCUSSION THREAD

**Discussion by songjinlian** [Leon406/ToolsFx#265] [Q&A] [answered]
![图片](https://github.com/Leon406/ToolsFx/assets/101912268/b7452724-807c-4f49-b94d-33cfa8eaee7f)

结果不应该是负数

**Comment by Leon406** [comment, marked as answer]
感谢反馈，github action已 构建

![image](https://github.com/Leon406/ToolsFx/assets/26186607/98af624d-ea03-476c-a5f9-e1405557ff73)
